pages/evm/get-update-fee.mdx

```diff
@@ -11,7 +11,7 @@ import { Tab, Tabs } from "nextra-theme-docs";
 
 Get the fee required to update the on-chain price feeds with the provided `updateData`.
 The returned number of wei should be sent as the transaction value when calling [updatePriceFeeds](update-price-feeds).
-The `updateData` can be retrieved from the price service API.
+The `updateData` can be retrieved from a Hermes instance.
```

Comment:
I think a bunch of the places in this PR where you say "Hermes instance" would benefit if the phrase indicated that this was some sort of web2 API. Some readers will read e.g.. this page of the docs without reading other pages. If that happens, they're going to be wondering "what the hell is hermes". I think "Hermes web API" or like "Hermes RPC" would be clearer. (You should pick a standard phrase for this and use it throughout) 

Also this can link to the hermes API docs now